### ISSUE TITLE
download-url downloads are incomplete from COINS

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#4042]
Initially reported in https://github.com/datalad/datalad/issues/3#issuecomment-575208711

**Comment by TobiasKadelka** [comment]
<details><summary>DataLad 0.12.0 WTF (configuration, datalad, dataset, dependencies, environment, extensions, git-annex, location, metadata_extractors, python, system)</summary>

# WTF
## configuration <SENSITIVE, report disabled by configuration>
## datalad 
  - full_version: 0.12.0
  - version: 0.12.0
## dataset 
  - id: afa5e920-1d7e-11ea-a633-b4969157768c
  - metadata: <SENSITIVE, report disabled by configuration>
  - path: /data/project/rehab_eNKI/eNKI_data
  - repo: AnnexRepo
## dependencies 
  - appdirs: 1.4.3
  - boto: 2.44.0
  - cmd:annex: 7.20190819+git2-g908476a9b-1~ndall+1
  - cmd:bundled-git: 2.20.1
  - cmd:git: 2.20.1
  - cmd:system-git: 2.24.0
  - cmd:system-ssh: 7.9p1
  - exifread: 2.1.2
  - git: 2.1.11
  - gitdb: 2.0.5
  - humanize: 0.5.1
  - iso8601: 0.1.11
  - keyring: 17.1.1
  - keyrings.alt: 3.1.1
  - msgpack: 0.5.6
  - mutagen: 1.40.0
  - requests: 2.21.0
  - tqdm: 4.28.1
  - wrapt: 1.10.11
## environment 
  - GIT_PYTHON_GIT_EXECUTABLE: /usr/lib/git-annex.linux/git
  - LANG: en_US.UTF-8
  - LANGUAGE: en_US.UTF-8
  - LC_ALL: en_US.UTF-8
  - LC_CTYPE: en_US.UTF-8
  - PATH: /home/tkadelka/.dotfiles/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/games
## extensions 
  - inm7: 
    - description: INM7 utilities
    - entrypoints: 
    - load_error: None
    - module: inm7_datalad
    - version: 0.5+10.g5f93ec5
  - ria: 
    - description: Helper for the remote indexed archive (RIA) special remote
    - entrypoints: 
      - ria_remote.export_archive.ExportArchive: 
        - class: ExportArchive
        - load_error: None
        - module: ria_remote.export_archive
        - names: 
          - ria-export-archive
          - ria_export_archive
    - load_error: None
    - module: ria_remote
    - version: 0.6+6.ge755961
## git-annex 
  - build flags: 
    - Assistant
    - Webapp
    - Pairing
    - S3
    - WebDAV
    - Inotify
    - DBus
    - DesktopNotify
    - TorrentParser
    - MagicMime
    - Feeds
    - Testsuite
  - dependency versions: 
    - aws-0.20
    - bloomfilter-2.0.1.0
    - cryptonite-0.25
    - DAV-1.3.3
    - feed-1.0.0.0
    - ghc-8.4.4
    - http-client-0.5.13.1
    - persistent-sqlite-2.8.2
    - torrent-10000.1.1
    - uuid-1.3.13
    - yesod-1.6.0
  - key/value backends: 
    - SHA256E
    - SHA256
    - SHA512E
    - SHA512
    - SHA224E
    - SHA224
    - SHA384E
    - SHA384
    - SHA3_256E
    - SHA3_256
    - SHA3_512E
    - SHA3_512
    - SHA3_224E
    - SHA3_224
    - SHA3_384E
    - SHA3_384
    - SKEIN256E
    - SKEIN256
    - SKEIN512E
    - SKEIN512
    - BLAKE2B256E
    - BLAKE2B256
    - BLAKE2B512E
    - BLAKE2B512
    - BLAKE2B160E
    - BLAKE2B160
    - BLAKE2B224E
    - BLAKE2B224
    - BLAKE2B384E
    - BLAKE2B384
    - BLAKE2BP512E
    - BLAKE2BP512
    - BLAKE2S256E
    - BLAKE2S256
    - BLAKE2S160E
    - BLAKE2S160
    - BLAKE2S224E
    - BLAKE2S224
    - BLAKE2SP256E
    - BLAKE2SP256
    - BLAKE2SP224E
    - BLAKE2SP224
    - SHA1E
    - SHA1
    - MD5E
    - MD5
    - WORM
    - URL
  - local repository version: 5
  - operating system: linux x86_64
  - remote types: 
    - git
    - gcrypt
    - p2p
    - S3
    - bup
    - directory
    - rsync
    - web
    - bittorrent
    - webdav
    - adb
    - tahoe
    - glacier
    - ddar
    - git-lfs
    - hook
    - external
  - supported repository versions: 
    - 5
    - 7
  - upgrade supported from repository versions: 
    - 0
    - 1
    - 2
    - 3
    - 4
    - 5
    - 6
  - version: 7.20190819+git2-g908476a9b-1~ndall+1
## location 
  - path: /data/project/rehab_eNKI/eNKI_data
  - type: dataset
## metadata_extractors 
  - annex: 
    - load_error: None
    - module: datalad.metadata.extractors.annex
    - version: None
  - audio: 
    - load_error: None
    - module: datalad.metadata.extractors.audio
    - version: None
  - datacite: 
    - load_error: None
    - module: datalad.metadata.extractors.datacite
    - version: None
  - datalad_core: 
    - load_error: None
    - module: datalad.metadata.extractors.datalad_core
    - version: None
  - datalad_rfc822: 
    - load_error: None
    - module: datalad.metadata.extractors.datalad_rfc822
    - version: None
  - exif: 
    - load_error: None
    - module: datalad.metadata.extractors.exif
    - version: None
  - frictionless_datapackage: 
    - load_error: None
    - module: datalad.metadata.extractors.frictionless_datapackage
    - version: None
  - image: 
    - load_error: None
    - module: datalad.metadata.extractors.image
    - version: None
  - xmp: 
    - load_error: No module named 'libxmp' [xmp.py:<module>:20]
    - module: datalad.metadata.extractors.xmp
## python 
  - implementation: CPython
  - version: 3.7.3
## system 
  - distribution: debian/10.2
  - encoding: 
    - default: utf-8
    - filesystem: utf-8
    - locale.prefered: UTF-8
  - max_path_length: 290
  - name: Linux
  - release: 4.19.0-6-amd64
  - type: posix
  - version: #1 SMP Debian 4.19.67-2+deb10u1 (2019-09-20)
</details>

**Comment by yarikoptic** [comment]
Hi @TobiasKadelka . Could you try again with current master or maint.  There were some fixups to downloaders recently, may be that would help... if not -- uff... I even re-registered on coins, tried to find eNKI but failed... could you guide me how to get to it @TobiasKadelka how to get to it so I could reproduce myself?

**Comment by TobiasKadelka** [comment]
Actually, this will be problematic. I had troubles with downloading the data last year (tried all the ways they were describing), so I wrote them a mail to their support. The person told me, that they were working on a better solution, but sent me temporary activated (for a week) download-URLs.
I looked at the website again, it does not look like they updated the download process so far. So I don't know, if there is a repruduceable way do obtain the enki data at the moment...
